google/resource_kms_crypto_key.go

```diff
@@ -0,0 +1,187 @@
+package google
+
+import (
+	"fmt"
```

Comment:
You should use GoImports (Really Gometalinter) to order these. Stdlib and 3rd party imports should not be mixed